src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -180,7 +183,11 @@ function OptionRowLHN(props) {
                             e.preventDefault();
                         }}
                         testID={optionItem.reportID}
-                        onSecondaryInteraction={(e) => showPopover(e)}
+                        onSecondaryInteraction={(e) => {
+                            showPopover(e);
+                            // Composer blur when opening context menu
```

Comment:
```suggestion
                            // Ensure that we blur the composer when opening context menu, so that only one component is focused at a time
```